README.md

```diff
@@ -33,6 +33,11 @@ We need to revamp the website and get full control of the 091labs.com domain.
 
 We need media packs to hand out.
 
+We also need basic elements of branding in an easily consumable form:
+ - Logo files
```

Comment:
It's possible that @darrenkearney may have a backup of these from back in the day?

Reply:
I think we have the png of the logo, we need slide templates, video templates, etc - enough that any new project can be trivially branded from day 1

Reply:
Hey folks, what's all this then? Hehe. Fun to see something happening here. I'll have a look tomorrow to see if I have a backup somewhere. My gut tells me yes, and that it's an Adobe illustrator file containing logo vectors.

Reply:
Yep. It's an old backup and the files are disorganized. I don't know if this has the exact version chosen by 091Labs in the end, but it has the logo device in a vector and some simple colour and layout variations. Send me a message and I'll give you the link.